requirements.txt

```diff
@@ -8,6 +8,7 @@ numpy
 opencv-python
 pandas
 pillow
+pre-commit
```

Comment:
Please revert.  pre-commit is not a runtime dependency.  It is a development dependency
```suggestion
```